scripts/models/enums.js

```diff
@@ -31,14 +31,14 @@ app.Enumeration.prototype.getName = function (value) {
 };
 
 
-app.Enums = { };
+app.Enums = {};
 
-app.Enums.NotificationType = new app.Enumeration({ Debug: 0, Notify: 1, Warning: 2, Error: 3 });
+app.Enums.NotificationType = new app.Enumeration({Debug: 0, Notify: 1, Warning: 2, Error: 3});
 
-app.Enums.UserStatus = new app.Enumeration({ Active: 0, LockedOut: 1, Disabled: 2, Deleted: 3 });
+app.Enums.UserStatus = new app.Enumeration({Active: 0, LockedOut: 1, Disabled: 2, Deleted: 3});
 
-app.Enums.UserRole = new app.Enumeration({ Administrator: 0, Client: 1 });
+app.Enums.UserRole = new app.Enumeration({Administrator: 0, Client: 1});
 
-app.Enums.AccessKeyType = new app.Enumeration({ Default: 0, Session: 1, Oauth: 2 });
+app.Enums.AccessKeyType = new app.Enumeration({Default: 0, Session: 1});
```

Comment:
Java-server auth moved from accessKey to JWT. Please, check if any other accessKey authorization is still present and change it to JWT.